easybuild/easyconfigs/p/Pychopper/Pychopper-0.4.0-intel-2018b-Python-3.7.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonPackage'
+
+name = 'Pychopper'
+version = '0.4.0'
+versionsuffix = '-Python-3.7.0'
```

Comment:
@HPC-UniOldenburg the 2018b toolchain generation requires Python-3.6.6 when used as a dependency